app/assets/javascripts/dul-arclight.js

```diff
@@ -274,10 +274,12 @@ Blacklight.onLoad(function () {
 
   /* ================================================ */
   /* Reload DUL masthead when turbolinks is triggered */
+  /* Scroll to hash tag anchor after turbolinks load  */
   /* ================================================ */
 
   $(document).on('ready turbolinks:load', function() {
```

Comment:
Ooh, nice. The event handling conveniences are the thing I miss most about not using jquery.